cmake/Modules/Findleveldb.cmake

```diff
@@ -0,0 +1,42 @@
+# Script to find LevelDB
+#
+# Sets the following variables:
+#
+#     leveldb_FOUND
+#     leveldb_LIBRARY
+#     leveldb_INCLUDE_DIR
+#
+# And the following imported targets:
+#
+#     leveldb::leveldb
```

Comment:
What about making the target & file name follow the official naming (LevelDB)?

as for the options you have free choice right? because keeping them uppercase would keep old build scripts working as before.

Reply:
The target naming was intentional to match the targets these projects export in their master branches, so that in the long term we can switch to using those projects' configs without having to update the target names all over the place. Good point about the build scripts. The options should stay the same as before, like you said.